pyaedt/hfss.py

```diff
@@ -6158,7 +6185,8 @@ def wave_port(
         reference : int, str, list or :class:`pyaedt.modeler.cad.object3d.Object3d`
             Ending object for the integration line or reference for Terminal solution. Can be multiple objects.
         create_port_sheet : bool, optional
-            Whether to create a port sheet or use given start_object as port shee.
+            Whether to create a port sheet or use given start_object as the surface to create
+            the port.
         create_pec_cap : bool, False
             Whether to create a port cap or not.
```

Comment:
```suggestion
            Whether to create a port cap.
```
I have to talk to Maxime. Should we move to defining the default after the data type in new code and gradually start updating the existing code?